app/components/AuthRequest/AuthRequest.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
```

Comment:
This, as part of SAFE, needs to originate from the extensions. There's a couple of example APIs if we need to create a new component.

The simpler thing though, may be to just make the Notifier more generically expandable, rather than creating a new component...